products/ocp4/profiles/cis.profile

```diff
@@ -25,6 +25,8 @@ description: |-
 
     This profile is applicable to OpenShift versions 4.10 and greater.
 
+extends: cis-1.4.0
+
 filter_rules: '"ocp4-node" not in platforms and "ocp4-master-node" not in platforms and "ocp4-node-on-sdn" not in platforms and "ocp4-node-on-ovn" not in platforms'
 
 selections:
```

Comment:
Remove these selections since they're going to be inherited from the extended profile.